packages/web3-shared/evm/hooks/useERC721TokenDetailedOwnerList.ts

```diff
@@ -85,75 +81,15 @@ async function getERC721TokenDetailedOwnerListFromChain(
 
     if (!isEnumerable || !balance) return []
 
-    const allRequest = Array.from({ length: min([Number(balance), queryLimit])! }).map(async (_v, i) => {
-        const tokenId = await safeNonPayableTransactionCall(
-            erc721TokenContract.methods.tokenOfOwnerByIndex(owner, i + offset),
-        )
+    const allRequest = Array.from({ length: Number.parseInt(balance, 10) }).map(async (_v, i) => {
```

Comment:
`Array.from` has a second argument that works like `Array#map`, you can combine the `.map` into it.
```suggestion
    const allRequest = Array.from({ length: Number.parseInt(balance, 10) }, async (_, i) => {
```